### PR TITLE
Fix link to "Officially supported languages and platforms"

### PR DESCRIPTION
**Pull request by georgeh**
I found some link rot while reading Officially Supported Languages

### CODE REVIEW

**Comment by yijiem** [comment]
CC @thisisnotapril 

This seems to be a good doc fix.

**Comment by dfawley** [comment]
@chalin can you take a look, please?

File: content/en/docs/what-is-grpc/faq.md [hunk 1]
```diff
@@ -31,9 +31,7 @@ Google has been using a lot of the underlying technologies and concepts in gRPC
 
 ### Which programming languages are supported?
 
-See [Officially supported languages and platforms][].
-
-[Officially supported languages and platforms]: /about/#officially-supported-languages-and-platforms
+See [Officially supported languages and platforms](https://grpc.io/docs/languages/).
```
Comment (ejona86) [inline content/en/docs/what-is-grpc/faq.md:34]: Looks like https://grpc.io/docs/#official-support is the 1:1 replacement for this link, as the table moved. Both links have the same list of languages, but the table has details about the platforms which is part of the link text. But the table doesn't have links to the documentation per-language.

I think either one is better than what we have. Would you mind either changing to link text to better match the url "See \[Supported languages\](...)" or leave the text the same and change the url to https://grpc.io/docs/#official-support ?

Comment (chalin) [inline content/en/docs/what-is-grpc/faq.md:34]: How about:

> For officially supported languages and platforms, see [Official support](https://grpc.io/docs/#official-support).

Linking to https://grpc.io/docs/#official-support.

**Comment by easwars** [comment]
Ping @georgeh

**Comment by georgeh** [comment]
This is a drive-by PR to fix a 404 link, feel free to merge or close.On Feb 15, 2023, at 10:15 AM, Easwar Swaminathan ***@***.***> wrote:﻿
Ping @georgeh

**Comment by ANA885874** [comment]
👍

**Comment by drfloob** [comment]
Commenting more generally: This PR appears to have been abandoned. Shall we fix it and land it in another PR? @ejona86

**Review by ejona86 — APPROVED** [review]

**Pull request merged**